docs/reference/eql/eql.asciidoc

```diff
@@ -608,16 +621,15 @@ GET /my-index-000001/_eql/search
 
 By default, EQL search requests are synchronous and wait for complete results
 before returning a response. However, complete results can take longer for
-searches across <<frozen-indices,frozen indices>> or
-<<modules-cross-cluster-search,multiple clusters>>.
-
-To avoid long waits, run an async EQL search. Set the
-`wait_for_completion_timeout` parameter to a duration you'd like to wait for
-synchronous results.
+searches across large data sets, <<data-tiers,cold>> or <<data-tiers,frozen>>
+data, or <<modules-cross-cluster-search,multiple clusters>>.
```

Comment:
Technically, this would still apply to frozen indices too, but that functionality is outdated with frozen tiers. See https://github.com/elastic/elasticsearch/issues/70192.